config/jobs/kubernetes/cloud-provider-aws/cloud-provider-aws-presubmit.yaml

```diff
@@ -17,10 +19,10 @@ presubmits:
         resources:
           limits:
             cpu: 2
-            memory: 4Gi
+            memory: 6Gi
```

Comment:
let's start here to unblock, but I meant 6Gi *per core* :-)

Reply:
we later [aligned in Slack ](https://kubernetes.slack.com/archives/C7J9RP96G/p1754584637214039?thread_ts=1754505741.634999&cid=C7J9RP96G)that we'll not lowered existing CPU =]